### PR TITLE
Configure prompts through files - [ENG 510]

### PR DESCRIPTION
**Pull request by varunshankar**
**Proposed changes**:
- Solves task: https://rasahq.atlassian.net/browse/ENG-510
- Components modified: LLMCommandGenerator

**Status (please check what you already did)**:
- [x] added some tests for the functionality
- [ ] updated the documentation
- [ ] updated the changelog (please check [changelog](https://github.com/RasaHQ/rasa/tree/main/changelog) for instructions)
- [x] reformat files using `black` (please check [Readme](https://github.com/RasaHQ/rasa#code-style) for instructions)

### CODE REVIEW

**Review by twerkmeister — COMMENTED** [review]
Hey Varun, two points:
* the other components are gonna be taken care of in rasa-plus, right? 
* Could you also add a test for the new functionality for the command generator here

File: data/test_prompt_templates/test_prompt.jinja2 [hunk 1]
```diff
@@ -0,0 +1 @@
+This is a test prompt.
```
Comment (tmbo) [inline data/test_prompt_templates/test_prompt.jinja2:1]:
```suggestion
This is a test prompt.

```

File: rasa/shared/utils/llm.py [hunk 2]
```diff
@@ -208,3 +209,22 @@ def embedder_factory(
         return embeddings_cls(**parameters)
     else:
         raise ValueError(f"Unsupported embeddings type '{typ}'")
+
+
+def get_prompt_template(
+    default_prompt_template: Text, jinja_file_path: Optional[Text]
```
Comment (tmbo) [inline rasa/shared/utils/llm.py:215]: STY: I'd suggest changing the parameter order, I am used to defaults being the last / later argument.

File: tests/cdu/generator/test_llm_command_generator.py [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+def resource() -> Resource:
+    return Resource(uuid.uuid4().hex)
+
+
+@pytest.fixture(scope="session")
+def model_storage(tmp_path_factory: TempPathFactory) -> ModelStorage:
+    return LocalModelStorage(tmp_path_factory.mktemp(uuid.uuid4().hex))
+
+
+@pytest.mark.parametrize(
```
Comment (tmbo) [inline tests/cdu/generator/test_llm_command_generator.py:25]: STY: I'd split this up into separate tests. keeps tests simpler and easier to reason about

**Review by tmbo — APPROVED** [review]

**Pull request merged**